sunpy/net/dataretriever/sources/eve.py

```diff
@@ -39,35 +42,34 @@ def _get_url_for_timerange(self, timerange, **kwargs):
 
         Parameters
         ----------
-        timerange: sunpy.time.TimeRange
+        timerange: `sunpy.time.TimeRange`
             time range for which data is to be downloaded.
 
         Returns
         -------
         urls : list
             list of URLs corresponding to the requested time range
         """
-        days = timerange.get_dates()
-        urls = []
-        for day in days:
-            urls.append(self._get_url_for_date(day, **kwargs))
-        return urls
-
-    def _get_url_for_date(self, date, **kwargs):
-        """
-        Return URL for corresponding date.
-
-        Parameters
-        ----------
-        date : Python datetime object
-
-        Returns
-        -------
-        URL : string
-        """
-        base_url = 'http://lasp.colorado.edu/eve/data_access/evewebdata/quicklook/L0CS/SpWx/'
-        return urljoin(base_url,
-                                date.strftime('%Y/%Y%m%d') + '_EVE_L0CS_DIODES_1m.txt')
+        baseurl = ('http://lasp.colorado.edu/eve/data_access/evewebdata/quicklook/'
```

Comment:
Here, there's a baseurl definition, and also in the next function. Wouldn't it be good to have a baseurl() function and define each baseurl once ?